CHANGELOG.md

```diff
@@ -13,10 +13,13 @@ Copy and pasting the git commit messages is __NOT__ enough.
 ### Changed
 - Changed waypoints in sumo maps to use more incoming lanes into junctions.
 - Increased the cutoff radius for filtering out waypoints that are too far away in junctions in sumo maps.
+- Made Envision dependencies optional. Use `pip install -e .[envision]` to install them.
```

Comment:
Envision does not appear to be optional since files like `agent_manager.py` appear to still import from envision.

Reply:
OK.